README.md

```diff
@@ -153,6 +153,36 @@ given buckets** before populating them with test fixture data, please ensure the
 
     tests/fixtures/populate.py --s3-bucket $DSS_S3_BUCKET_TEST_FIXTURES --gs-bucket $DSS_GS_BUCKET_TEST_FIXTURES
 
+#### Set up the whitelist
+
+1. Follow [instructions here](https://github.com/ucsc-cgp/bouncer#setting-up-the-whitelist) to set up the whitelist.
+   The convention for naming the whitelist is `commons/DSS_DEPLOYMENT_STAGE/whitelist`, substituting `DSS_DEPLOYMENT_STAGE`
+   for its value.
+
+2. Then you will need to adjust your environment. Copy this snippet and add it to your `environment.local`
+
+   ```
+   EXPORT_ENV_VARS_TO_LAMBDA_ARRAY+=(
+       EMAIL_WHITELIST_NAME
+       )
+   ```
+
+   Also add the variable `EMAIL_WHITELIST_NAME` to your `environment.local` with the value as
+   the name from the previous step. For example one might add
+
+   ```
+   EMAIL_WHITELIST_NAME=commons/DSS_DEPLOYMENT_STAGE/whitelist
+   ```
+
+   Then run
+   ```
+   source environment
+   ```
+
+3. Since you are using service account credentials for you local deployment you will need to use the email listed in your
+   `gcp-credentials.json` which is long and contains your GCP project ID among other details. This email is also viewable in
```

Comment:
Please change:
```
listed in your `gcp-credentials.json`
```
to:
```
included in the value of your `$GOOGLE_APPLICATION_CREDENTIALS`
```

Reply:
As we discussed, I'll leave it as `gcp-credentials.json` since that's what the README tells you to make earlier and because the environment variable may not be set.